node/flows.re

```diff
@@ -412,3 +414,51 @@ let request_withdraw_proof = (state, ~hash) =>
   };
 let request_ticket_balance = (state, ~ticket, ~address) =>
   state.Node.protocol.ledger |> Ledger.balance(address, ticket);
+
+let trusted_validators_membership =
+    (
+      ~file,
+      ~persist:
+         (list(Trusted_validators_membership_change.t), ~file: string) =>
+         Lwt.t(unit),
+      state,
+      update_state,
+      request,
+    ) => {
+  open Networking.Trusted_validators_membership_change;
+  let {signature, payload: {address, action} as payload_} = request;
+  switch (
+    payload_ |> payload_to_yojson |> Yojson.Safe.to_string |> BLAKE2B.of_string
+  ) {
+  | Some(payload_hash) =>
+    if (payload_hash |> Signature.verify(~signature)) {
+      let new_validators =
+        switch (action) {
+        | Add =>
+          Trusted_validators_membership_change.Set.add(
+            {action: Add, address},
+            state.Node.trusted_validator_membership_change,
+          )
+        | Remove =>
+          Trusted_validators_membership_change.Set.remove(
+            {address, action: Remove},
+            state.Node.trusted_validator_membership_change,
+          )
+        };
+      let _: State.t =
+        update_state({
+          ...state,
+          trusted_validator_membership_change: new_validators,
+        });
+      let.await () =
+        persist(
+          ~file,
+          new_validators |> Trusted_validators_membership_change.Set.elements,
+        );
+      Result.ok() |> Lwt.return;
```

Comment:
```suggestion
      await(Ok());
```

Reply:
I'm committing this. But, `await` seems like a very bad name creating Lwt monads.